universal-login-example/src/relayer/utils.js

```diff
@@ -14,4 +14,13 @@ const waitForContractDeploy = async (providerOrWallet, contractJSON, transaction
   return new Contract(receipt.contractAddress, abi, providerOrWallet);
 };
 
-export {waitForContractDeploy};
+const waitToBeMined = async (providerOrWallet, transactionHash, tick = 1000) => {
```

Comment:
just `provider`, no need for `providerOrWallet`

Reply:
✅